### PR TITLE
Added DockQ

### PR DESCRIPTION
**Pull request by peterbygrave**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by peterbygrave** [comment]
@conda-forge/bioconda-recipes Looking at adding this recipe to conda-forge to help build out the build variants across Python and numpy versions, given this package is compiled and cython'd code is numpy based. Happy to hear any other advice too 🙂

**Comment by peterbygrave** [comment]
@conda-forge/help-python-c ready for review

Thank you

**Comment by peterbygrave** [comment]
@conda-forge/help-python ready for review

File: recipes/DockQ/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/dockq-${{ version }}.tar.gz
+  sha256: 50c4e2b4bced3bf865b12061ec0b56e23de1383dc70b445441848224f6c72c0d
+
+build:
+  number: 0
+  skip: match(python, "<3.8")
```
Comment (ocefpaf) [inline recipes/DockQ/recipe.yaml:15]: Conda-forge's min supported python is 3.9, you don't need this.

```suggestion
```

**Pull request merged**